src/entities/Route/types.test.ts

```diff
@@ -60,9 +59,12 @@ const invalidOriginCases = [
   'https://now.sh',
   'https://subdomain.now.sh',
   'http://manaland.cn',
+  'https://manaland.cn',
+  'https://subdomain.manaland.cn',
   'http://subdomain.manaland.cn',
   'http://other.domain',
   'https://other.domain',
+  'http://new-landing-page.landing-9sh.pages.dev/',
```

Comment:
please, replace it for something more generic like 
```suggestion
  'http://custom-build.custom-page-1a2b.pages.dev/',
```